src/pkrd/views/pkx.rs

```diff
@@ -31,6 +31,26 @@ pub fn run_view(
         y += 12;
         let tsv_text = &alloc::format!("TSV: {:04}", pkx.tsv());
         screen.draw_string(&white, tsv_text, x, y)?;
+
+        y += 12;
+        let nature_text = &alloc::format!("Nature: {}", pkx.nature().to_string());
+        screen.draw_string(&white, nature_text, x, y)?;
+
+        y += 12;
+        let ability_text = &alloc::format!(
+            "Ability: {} ({})",
+            pkx.ability().to_string(),
```

Comment:
At some point we'll want to trim this, although we can deal with that later since we're still not fully sure what the dimensions of each view will be.